src/test/java/org/opentripplanner/ConstantsForTests.java

```diff
@@ -135,13 +143,23 @@ public static NetexBundle createMinimalNetexNordicBundle() {
 
   public static NetexBundle createMinimalNetexEpipBundle() {
     var buildConfig = createNetexEpipBuilderParameters();
+    List<String> fileList = new ArrayList<>();
+    generateFileList(new File(NETEX_EPIP_DATA_DIR), fileList, NETEX_EPIP_DATA_DIR);
+    zipIt(NETEX_EPIP_DIR + NETEX_EPIP_FILENAME, NETEX_EPIP_DATA_DIR, fileList);
+
     var netexZipFile = new File(NETEX_EPIP_DIR, NETEX_EPIP_FILENAME);
 
     var dataSource = new ZipFileDataSource(netexZipFile, FileType.NETEX);
```

Comment:
We tried using the `DirectoryDataSource` but noticed that all sub directories are skipped in the `content()` method. Since our dataset contains sub directories, we would need to either change the data including the build config or the method. Is it intentional that sub directories are skipped? And if so why?

Reply:
That's a really good question. Let's get to the bottom of during today's meeting.